Exploratory_Data_Analysis/Principles_of_Analytic_Graphs/lesson

```diff
@@ -126,7 +126,7 @@
   Output: The fourth principle of analytic graphing involves integrating evidence. This means not limiting yourself to one form of expression. You can use  words, numbers, images as well as diagrams. Graphics should make use of many modes of data presentation. Remember, "Don't let the tool drive the analysis!"
 
 - Class: figure
-  Output: To show you what we mean, here's an example of a figure taken from a paper published in the Journal of the AMA. It shows the relationship between pollution and hospitalization of people with heart disease. As you can see, it's a lot different from our previous plots. The solid circles in the center portion indicate point estimates of percentage changes in hospitalization rates for different levels of pollution. The lines through the circles indicate confidence intervals associated with these estimates. (You'll learn more about confidence intervals in another great course, this one on statistical inference.) 
```

Comment:
This line is referencing a separate course, so referring to the statistical inference course as "this course" is inaccurate.